it/utils/ViewHelpers.scala

```diff
@@ -24,6 +24,7 @@ import org.scalatest.wordspec.AnyWordSpec
 import play.api.http.HeaderNames
 import play.api.libs.ws.{BodyWritable, WSClient, WSResponse}
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import sttp.model.Method.POST
```

Comment:
Sorry I missed that before approving. This import can be removed

Reply:
Done